.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing to `<media-chrome>`
+
+- [Questions](#questions)
+- [Bugs and Issues](#issues)
+- [Documentation Updates](#documentation)
+- [Feature Requests](#features)
+- [Submitting a Pull Request](#pull-requests)
+
+## <a name="questions">Questions</a>
```

Comment:
Github already does some link anchoring for headings, if that's enough for this use case.

Reply:
I think the only benefit here over using github's auto-linking is that we can name these ourselves for having it be easier to write the TOC above.